app/views/reports/regions/details.html.erb

```diff
@@ -67,16 +68,18 @@
               Total
             </td>
             <td class="row-total ta-center">
-              <%= number_with_delimiter(dash_if_zero(@dashboard_analytics.sum { |_, row| row[:total_registered_patients] || 0 })) %>
+              <%= number_or_dash_with_delimiter(@dashboard_analytics.sum { |_, row| row[:total_registered_patients] || 0 }) %>
             </td>
-            <% dates_for_periods(@period.type, 6, include_current_period: @show_current_period).each do |date| %>
+            <% @period_range.each do |period| %>
               <td class="row-total ta-center">
-                <%= number_with_delimiter(dash_if_zero(analytics_totals(@dashboard_analytics, :registered_patients_by_period, date))) %>
+                <%= number_or_dash_with_delimiter(@repository.registration_counts[@region.region.slug][period]) %>
               </td>
             <% end %>
-            <% dates_for_periods(@period.type, 6, include_current_period: @show_current_period).each do |date| %>
+            <% @period_range.each do |period| %>
               <td class="row-total ta-center">
-                <%= number_with_delimiter(dash_if_zero(analytics_totals(@dashboard_analytics, :follow_up_patients_by_period, date))) %>
+                <%= number_with_delimiter(dash_if_zero(analytics_totals(@dashboard_analytics, :follow_up_patients_by_period, period.value))) %>
+                ||
+                <%= number_or_dash_with_delimiter(@repository.hypertension_follow_ups(group: "blood_pressures.user_id")[@region.region.slug][period]) %>
```

Comment:
I was verifying numbers locally with lots of manual QA by displaying both -- I'l remove the legacy numbers before merging this.

Reply:
We're directly grabbing data from the repository here, whereas we're delegating that to the "analytics query" for registration info. Is that inconsistent, using the repository directly as well as through that query class? Can we add the necessary `@dashboard_analytics.hypertension_follow_ups(group: :something)` to the analytics query classes and keep our access consistently through `@dashboard_analytics`?